projects/eslint-plugin/rules/portal/lib/rules/no-global-fetch.js

```diff
@@ -16,10 +16,12 @@ module.exports = {
 
 		const isValidDefaultImport = (node) => {
 			return (
-				node.source.value.endsWith('/fetch.es') &&
-				node.specifiers.find(
-					(specifier) => specifier.type === 'ImportDefaultSpecifier'
-				)
+				node.source.value.endsWith('/fetch') ||
+				(node.source.value.endsWith('/fetch.es') &&
+					node.specifiers.find(
+						(specifier) =>
+							specifier.type === 'ImportDefaultSpecifier'
+					))
```

Comment:
I think the parenthesis needs to move here so that the OR is for the endsWith and the AND is for the find

```suggestion
				(node.source.value.endsWith('/fetch') ||
					node.source.value.endsWith('/fetch.es')) &&
				node.specifiers.find(
					(specifier) =>
						specifier.type === 'ImportDefaultSpecifier'
				)
```

Reply:
Good catch, that was prettier that autofixed that in that specific manner